### PR TITLE
OOP Task

### PR DESCRIPTION
**Pull request by SiarheiTsytrykau**
Last.fm Web Services

### CODE REVIEW

File: LastFm/js/LastFm.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+function LastFm(api_key) {
+  this.apiKey = "322b365de048b251b67a31166b9529d0";
+  this.rootURL = "http://ws.audioscrobbler.com/2.0/";
+}
+
+LastFm.prototype.load = function(queryParam, callback) {
+  var url = this.rootURL + "?method=" + this.package + "." + queryParam + "&api_key=" + this.apiKey + "&format=json";
```
Comment (NikitaSedyx) [inline LastFm/js/LastFm.js]: please look at https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

File: LastFm/js/Runner.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+var cher = new Artist("cher");
+cher.getInfo(function(response){
+  console.log(response.artist);
+});
+cher.search(function(response){
+  console.log(response.results);
+});
+
+var cherBelive = new Album("cher", "Believe");
+cherBelive.getInfo(function(response){console.log(response.album)});
```
Comment (NikitaSedyx) [inline LastFm/js/Runner.js]: it needs some spaces

File: LastFm/index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        Author <a href="mailto:cpserg@gmail.com?Subject=JS%20training%20Last.fm Web Services" title="Send message">Siarhei Tsyrtykau</a>
+      </address>
+    </footer>
+    <script src="js/LastFm.js" defer></script>
+    <script src="js/Artist.js" defer></script>
+    <script src="js/Album.js" defer></script>
+    <script src="js/Chart.js" defer></script>
+    <script src="js/Runner.js" defer></script>
+  </body>
+</html>
```
Comment (NikitaSedyx) [inline LastFm/index.html]: eof

File: LastFm/index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  <body>
+    <header>
+      <h1>Last.fm Web Services</h1>
+    </header>
+    <footer>
+      <address>
+        Author <a href="mailto:cpserg@gmail.com?Subject=JS%20training%20Last.fm Web Services" title="Send message">Siarhei Tsyrtykau</a>
+      </address>
+    </footer>
+    <script src="js/LastFm.js" defer></script>
```
Comment (NikitaSedyx) [inline LastFm/index.html]: do you really need `defer` here? you don't have `async` attribute

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]
mostly fine, please look at comments

**Comment by SiarheiTsytrykau** [comment]
Comments fixed

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**